SharedCode/src/commonMain/kotlin/common.kt

```diff
@@ -51,4 +51,20 @@ fun getEpochFromUTC(s: String): Long {
 }
 fun getPrice(tickets: List<ApiResult.Journey.Ticket>): Int {
     return tickets.map { it.priceInPennies }.sum()
+}
+
+fun ApiResult.toTrainTimes(): TrainTimes{
+    val journeys = mutableListOf<TrainTimes.Journey>()
+    this.outboundJourneys.forEach { journey ->
+        journeys.plusAssign(TrainTimes.Journey(
+            getPrice(journey.tickets),
+            getEpochFromUTC(journey.departureTime),
+            getEpochFromUTC(journey.arrivalTime),
+            journey.legs.size-1
```

Comment:
I noticed a small discrepancy with journey.legs.size and journey.tickets.size which I will investigate now before I merge.

Reply:
That's interesting, let me know if you want me to take a look or anything

Reply:
Okay it is not the case as previously thought that `(number legs) - 1 = (number tickets) - 1`. But this problem is not related to the purpose of this PR so I shall merge.

Reply:
You're right, one ticket can do multiple legs